THANKS.txt

```diff
@@ -130,7 +130,9 @@ Jörg Dietrich for the k-sample Anderson Darling test.
 Blake Griffith for improvements to scipy.sparse.
 Andrew Nelson for scipy.optimize.differential_evolution.
 Brian Newsom for work on ctypes multivariate integration.
-Nathan Woods for advising on multivariate integration and building unit tests.
+Nathan Woods for work on multivariate integration. His work from 31 Aug 2015
+    to the present is subject to the disclaimer at:
+    https://github.com/losalamos/Gazebo/blob/master/LICENSE.
```

Comment:
Thinking about it some more, the link is really not OK like this. The LICENSE file it points to could be changed to something incompatible with SciPy's license without us noticing.

Commit message also doesn't work well. How about you add the full disclaimer to this file and above it say "copied from https://github.com/losalamos/Gazebo/blob/master/LICENSE on 28 Feb 2016".